iOS/Old UI/Settings/TrackersViewController.swift

```diff
@@ -173,7 +173,58 @@ extension TrackersViewController {
             }
             session.presentationContextProvider = self
             session.start()
+        } else if let tracker = tracker as? HostUserPassTracker {
+            let loginPopUp = UIAlertController(title: String(format: NSLocalizedString("%@_LOGIN_TRACKER", comment: ""), tracker.name),
```

Comment:
indentation for this should be like
```swift
let loginAlert = UIAlertController(
    title: ...,
    message: ...,
    ...
)
```

and say Alert instead of PopUp